mailbox/api/src/test/java/org/apache/james/mailbox/events/ErrorHandlingContract.java

```diff
@@ -161,12 +160,12 @@ default void exceedingMaxRetriesShouldStopConsumingFailedEvent() throws Exceptio
         eventBus().register(throwingListener, GROUP_A);
         eventBus().dispatch(EVENT, NO_KEYS).block();
 
-        Thread.sleep(getSpeedProfile().getLongWaitTime().toMillis());
-        int numberOfCallsAfterExceedMaxRetries = throwingListener.timeElapsed.size();
-        Thread.sleep(getSpeedProfile().getLongWaitTime().toMillis());
+        getSpeedProfile().shortWaitCondition()
+            .untilAsserted(() -> assertThat(throwingListener.executionCount()).isEqualTo(4));
```

Comment:
this change is not what the comment describe

Reply:
I likely messed up while rebasing, sorry.